config/task.config

```diff
@@ -0,0 +1,54 @@
+process {
+
+    withName: Cutadapt_1 {
+        ext.prefix = {"${meta.id}_1"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args =' -a "^TEST...TEST$" --discard-untrimmed '
+    }
+
+     withName: Cutadapt_2 {
+        ext.prefix =  {"${meta.id}_2"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args =' -a "^TEST...TEST$" --discard-untrimmed '
+    }
+
+    withName: Cutadapt {
+        publishDir = { "${params.outdir}/${meta.id}/1: Trimmed reads/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
Is `Trimmed reads` a directory name? Is so, please make it `Trimmed_reads`, spaces in paths are annoying when working in a terminal.